### ISSUE TITLE
Detailed Hardware specification & type of DNN used.

### ISSUE THREAD

**Issue by gambitier** [zoogzog/chexnet#2]
1.  I got that you used Tesla P100 GPU, but I just wanted to know what are its specifications e.g. memory, DDR4/DDR5, no. of cores etc.

2.  second, what was CPU specification?

3. Which Deep Neural Nets can be used to train images? Did you used ConvNets? and what do you think about using Capsule Neural Nets to train the model? will it improve model accuracy?

**Comment by zoogzog** [comment]
1&2 Here are the specifications of my machine: 2x[Xeon CPU E52603](https://ark.intel.com/products/92993/Intel-Xeon-Processor-E5-2603-v4-15M-Cache-1_70-GHz), 4x[Tesla P100 NVLINK](http://www.nvidia.com/object/tesla-p100.html) (3584 cores), RAM 256GB

The results I got when running on single GPU, by specifying CUDA_VISIBLE_DEVICES for the python script, thinking that it would load the GPU better than spreading in parallel with batch size = 16.
My latest tests showed that running in parallel on all 4 GPUs with batch size 256 allowed to increase processing speed (~ 3 times), without significant changes in accuracy.

3 My primary interest was implementing the [CheXNet](https://arxiv.org/abs/1711.05225), so I have  not done any tests with other networks yet. The authors of the paper, that presented (ChestX-ray)[https://arxiv.org/pdf/1705.02315.pdf] did some tests with ResNet, VGGN, AlexNet. So far, DenseNet architecture showed best results.

**Comment by gambitier** [comment]
Hey, thanks for your response, 
the hardware specs you specified seems costly, so how did you managed to get it? Did you used cloud services like aws, GCP etc? and what was the total cost required to train your model?
